Compiler/Node.py

```diff
@@ -134,7 +135,7 @@ def get_code(self, current_pointer, *args, **kwargs):
 
 
 class NodeUnaryPostfix(Node):
-    def __init__(self, ids_map_list, operation, literal, *kargs, **kwargs):
+    def __init__(self, ids_map_list, operation, literal):
```

Comment:
Good catch
Please also remove kargs, kwargs in NodeArrayElement constructor in line 204

Reply:
Should I also remove the ones in def `get_code()`?